test/OpenTelemetry.Tests/Metrics/MetricViewTests.cs

```diff
@@ -572,6 +572,77 @@ public void ViewToProduceCustomHistogramBound()
             Assert.Equal(boundaries.Length + 1, actualCount);
         }
 
+        [Theory]
+        [MemberData(nameof(MetricTestData.ValidHistogramMinMax), MemberType = typeof(MetricTestData))]
+        public void HistogramMinMax(double[] values, HistogramConfiguration histogramConfiguration, double expectedMin, double expectedMax)
+        {
+            using var meter = new Meter(Utils.GetCurrentMethodName());
+            var histogram = meter.CreateHistogram<double>("MyHistogram");
+            var exportedItems = new List<Metric>();
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddMeter(meter.Name)
+                .AddView(histogram.Name, histogramConfiguration)
+                .AddInMemoryExporter(exportedItems)
+                .Build();
+
+            for (var i = 0; i < values.Length; i++)
+            {
+                histogram.Record(values[i]);
+            }
+
+            meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+
+            var metricPoints = new List<MetricPoint>();
+            foreach (ref readonly var mp in exportedItems[0].GetMetricPoints())
+            {
+                metricPoints.Add(mp);
+            }
+
+            var histogramPoint = metricPoints[0];
+            var hasMin = histogramPoint.TryGetHistogramMin(out var min);
+            var hasMax = histogramPoint.TryGetHistogramMax(out var max);
+
+            Assert.True(hasMin);
+            Assert.True(hasMax);
+            Assert.Equal(expectedMin, min);
+            Assert.Equal(expectedMax, max);
+        }
+
+        [Theory]
+        [MemberData(nameof(MetricTestData.InvalidHistogramMinMax), MemberType = typeof(MetricTestData))]
+        public void HistogramMinMaxThrows(double[] values, HistogramConfiguration histogramConfiguration)
+        {
+            using var meter = new Meter(Utils.GetCurrentMethodName());
+            var histogram = meter.CreateHistogram<double>("MyHistogram");
+            var exportedItems = new List<Metric>();
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddMeter(meter.Name)
+                .AddView(histogram.Name, histogramConfiguration)
+                .AddInMemoryExporter(exportedItems)
+                .Build();
+
+            for (var i = 0; i < values.Length; i++)
+            {
+                histogram.Record(values[i]);
+            }
+
+            meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+
+            var metricPoints = new List<MetricPoint>();
+            foreach (ref readonly var mp in exportedItems[0].GetMetricPoints())
+            {
+                metricPoints.Add(mp);
+            }
+
+            var histogramPoint = metricPoints[0];
+
+            var ex = Assert.Throws<NotSupportedException>(() => histogramPoint.TryGetHistogramMin(out var min));
```

Comment:
this should not throw. Exports has to call TryGetHistogramMin for MetricType=Histogram, irrespective of it has min/max. The way exporters determine if min/max available is from the return value of TryGet.

Reply:
Yes, thank you!